### PR TITLE
Fix drone building on windows

### PR DESCRIPTION
**Pull request by thardeck**
This should prevent the following Drone Windows error when building Docker images:
Invoke-WebRequest : Access to the path 'C:\git.zip' is denied.

e.g. https://drone-publish.rancher.io/rancher/fleet/3178/3/2

We could most likely change the directory but we are moving away from Drone building short term so it is not worth it.

### CODE REVIEW

File: package/Dockerfile-windows.agent [hunk 1]
```diff
@@ -2,15 +2,15 @@ ARG SERVERCORE_VERSION
 FROM mcr.microsoft.com/windows/servercore:${SERVERCORE_VERSION}
 ARG RELEASES
 ARG VERSION
-USER ContainerUser
```
Comment (weyfonk) [inline package/Dockerfile-windows.agent:5]: Was this incompatible with `Invoke-WebRequest`? We added it as part of #2042.

Comment (weyfonk) [reply package/Dockerfile-windows.agent:5]: I see. Anyway, this is most likely a short-term fix as we may well end up cross-building Windows-based images from Linux hosts, meaning that download/unzip steps would happen from another image. We could then (but only then) keep `USER ContainerUser`.

TL;DR: this is fine, at least as a short-term fix. My first comment may have been a bit too hasty :þ

**Review by weyfonk — APPROVED** [review]

File: package/Dockerfile-windows.agent (hunk 1 above)
Comment (manno) [reply package/Dockerfile-windows.agent:5]: Maybe we can move it to the bottom? So the powershell commands are still allowed to write to `C:\`.. :shrug:
